docs/import/add_disaster.js

```diff
@@ -143,41 +157,43 @@ function updateAfterSort(ui) {
 
   // dumbly renumber all the layers. note: nth-child starts at 1, .index()
```

Comment:
nit: not so dumb anymore :) could stand a comment here explaining thinking

Reply:
Ya juuuuust missed it. Updated.